src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -382,6 +382,7 @@ class EmojiPickerMenu extends Component {
      */
     filterEmojis(searchTerm) {
         const normalizedSearchTerm = searchTerm.toLowerCase().trim();
+        this.emojiList.scrollToOffset({offset: 0, animated: false});
```

Comment:
```suggestion
        if (this.emojiList) {
            this.emojiList.scrollToOffset({offset: 0, animated: false});
        }
```
@stitesExpensify I think we only need to apply when `this.emojiList` is available, bcoz for no results found we are not rendering the `flatList` and in that case when we modify that search text it will give console error `Cannot read properties of null (reading 'scrollToOffset')`.

Reply:
Good shout, updated!